test/spec/protractor_sync_test.ts

```diff
@@ -88,6 +94,16 @@ describe('Protractor extensions', () => {
       testMultilineInput = elementSync.findVisible('textarea');
     }));
 
+    it('causes large heap snapshot on error', createTest(() => {
+      heapdump.writeSnapshot('/Users/mwhitsitt/heaps/beforeFailure.heapsnapshot');
+      polledExpect(() => { return false; }).toBe(true);
+    }));
+
+    it('causes same size heap snapshot on error', createTest(() => {
+      heapdump.writeSnapshot('/Users/mwhitsitt/heaps/beforeFailure.heapsnapshot');
+      expect(false).toBe(true);
+    }));
```

Comment:
this test case generates before and after heap snapshots that are both about 17MB in size